docs/Run-the-app-in-localhost.md

```diff
@@ -0,0 +1,23 @@
+# Run the app in localhost
+
+## Prerequisites
+
+- Java 17 or later
+
+## Steps to run the app in localhost
+
+1. Build sample project.
+
+    ```shell
+    ./mvnw clean package -DskipTests
+    ```
+
+2. Run sample project.
+
+    ```shell
+    ./mvnw spring-boot:run -f web/pom.xml
```

Comment:
we ran into this before, it's not going to work

Reply:
Fixed.